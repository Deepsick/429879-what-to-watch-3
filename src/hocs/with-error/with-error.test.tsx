import * as React from 'react';
import * as renderer from 'react-test-renderer';

import withError from './with-error';

interface MockComponentProps {
  children: React.ReactNode;
}

const MockComponent = (props: MockComponentProps) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

const MockComponentWrapped = withError(MockComponent);

it(`withError is rendered correctly`, () => {
  const tree = renderer.create(
      <MockComponentWrapped />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
