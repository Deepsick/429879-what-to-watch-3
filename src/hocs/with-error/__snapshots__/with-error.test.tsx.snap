// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withError is rendered correctly 1`] = `<div />`;
